python/cudf/cudf/tests/test_numerical.py

```diff
@@ -293,9 +293,12 @@ def test_to_numeric_downcast_string_int(data, downcast):
 @pytest.mark.parametrize(
     "data",
     [
+        [""],  # pure null strings
```

Comment:
this isn't a `null` string, it's an empty string. We should be careful to make sure to differentiate between the two.

Reply:
Yup David also mentioned the difference in the chat. That's a oversight on naming here.